### PR TITLE
Dashboard mockup with test elevation graph

### PR DESCRIPTION
**Pull request by KristinChengWu**
This is a bare boned dashboard that has tabs and cards for each graph. Just to show an example of a graph, the "Elevation" card in the "Route Map" tab graphs distance vs. elevation data from assets/wsc_elevation.csv. 

I didn't add the dash packages to the requirements.txt file. If you want to run it, pip install dash and dash-bootstrap-components and run the index.py file. Or you can message me on Slack to do a screenshare :)

### CODE REVIEW

**Review by mdshiozaki — APPROVED** [review]
Didn't want to ping all the graph1/graph2 if you do change it, but looks like a good start!

File: displays/dashboard/assets/style.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.index-tabs {
+    box-shadow: 2px 0px 2px darkgrey;
+    height: 100vh;
+    padding-top: 50px;
+}
+
+.index-tab{
+    background-color: var(--bg-color) !important;
```
[the remark this answers is not in the document]
Comment (KristinChengWu) [reply displays/dashboard/assets/style.css:33]: I think actually the default styles used by dash

**Comment by mdshiozaki** [comment]
@KristinChengWu can merge when you please, thanks!

**Comment by KristinChengWu** [comment]
> @KristinChengWu can merge when you please, thanks!

I don't think I have write access to this repository so I can't merge pull requests myself.

**Pull request merged**